apis/v0/doc.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:generate protoc -I../../protos/v0 --go_out=. --go-grpc_out=. --go_opt=paths=source_relative --go-grpc_opt=paths=source_relative jvs_request.proto jvs_service.proto cert_action_request.proto cert_action_service.proto
```

Comment:
I'm not sure I'd expect this in `doc.go`, but it's not a blocker.

Reply:
`generate.go`?

Reply:
Usually I see things kinds of things (and the package comment) in a file named after the package (v0.go in this case). Non-blocking and not super important.